idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -140,12 +138,18 @@ class PackedColumnData(UnitModelBlockData):
         default=250,
         domain=float,
         description="Specific surface area of packing (m^2/m^3)",
-        doc="Surface area of packing per unit volume of column(default= 250 m2/m3)"))
+        doc="Surface area of packing per unit volume of column("
+        "default= 250 m2/m3)"))
     CONFIG.declare("packing_void_fraction", ConfigValue(
         default=0.97,
         domain=float,
         description="Void fraction of the packing",
         doc="Packing porosity or void fraction (default= 0.97 )"))
+    CONFIG.declare("packing_channel_size", ConfigValue(
+        default=0.1,
+        domain=float,
+        description="Channel size of the packing",
+        doc="Packing channel size default = 0.1 m"))
     CONFIG.declare("fix_column_pressure", ConfigValue(
```

Comment:
Is this the same as `has_pressure_drop`? If so, we should use the standard name.